packages/server/postgres/queries/insertMeetingTemplate.ts

```diff
@@ -0,0 +1,18 @@
+import getRethink from '../../database/rethinkDriver'
+import MeetingTemplate from '../../database/types/MeetingTemplate'
+import getPg from '../getPg'
+
+const insertMeetingTemplate = async (meetingTemplate: MeetingTemplate) => {
+  const r = await getRethink()
+  const pg = getPg()
+  const {id, name, teamId, orgId, parentTemplateId, type} = meetingTemplate
+  await Promise.allSettled([
```

Comment:
the `allSettled` pattern seemed cleaner than a try/catch because we almost never return a value from a mutation. In Phase 3 when we stop writes to RethinkDB we can just delete a couple lines & we're done!

Reply:
-1 I'm not sure I can follow here. This would just silently ignore all errors, wouldn't it?

Reply:
We can't `Promise.all` because then it'll throw if either promise rejects. We can treat it as 2 separate promises, that's what I did in a previous commit:
 https://github.com/ParabolInc/parabol/pull/7679/commits/bd580a74ab72ad930454f1b06a7da189c4dcf75a#diff-b5b9e532d5e86ec912acda5064778d79f5c9ce4390f66acc77b5720882b5143aL8-L26
 
 I didn't like that very much because it means we have to create the rethink promise, then await the pg promise, then await the rethink promise to make sure they work in parallel. I thought that pattern might reduce readability for the team, but I'm OK with doing whatever makes sense!

Reply:
In cases where we expect postgres to fail (so not for inserts), we can use `Promise.allSettled` but need to verify and throw if rethink did throw an error. So adding a
```
  if (result.status === 'rejected') throw new Error(result.reason)
```
would work for me.

I wasn't so concerned about the performance around `MeetingTemplate` and would simply await both promises (rethink and postgres) one after the other for readability, but checking and throwing is cleanest I think (again only where we expect postgres to fail)

Reply:
that is super reasonable. will fix!

Reply:
one thing i realized-- rethinkdb doesn't throw if an update fails. it'll only throw if it can't reach the DB, or if we didn't write the query correctly, etc. Still nice to throw in those conditions, it'll just be pretty rare